ydb/core/persqueue/writer/writer.cpp

```diff
@@ -313,6 +319,14 @@ class TPartitionWriter: public TActorBootstrapped<TPartitionWriter>, private TRl
         cmd.SetForce(true);
 
         SetWriteId(request);
+        // While waiting for a response from KQP, the recording session could
+        // send the following messages. They are in the "Pending" queue and 
+        // the FirstWrite flag will be reset
+        if (Pending) {
```

Comment:
Why can some GetOwnership requests be with FirstWrite == false?

Reply:
Fixed
https://github.com/ydb-platform/ydb/pull/4758/commits/7b56087e43f8fe11e87371e5f3d79292aff5717a